README.md

```diff
@@ -52,7 +52,6 @@ write messages onto the queue:
 
 ```ruby
 message_data = {
-  "event_type" => "heartbeat",
   "data" => {
```

Comment:
> I think I could be beneficial to continue to conform to that even in the formatting of our messages. As a good start, "data" is a top level item in our JsonApi world.

Oh nice, I totally agree. I'm going to keep the top-level `data` key for now. Might be worth discussing a follow-up PR to ensure it conforms. 

Funny, I actually didn't think to add `data:` as a key because of JSON-API, but looks like it was a happy coincidence :)